### ISSUE TITLE
Compile for kernel 4.0.3 fails - struct video_device has no member named debug

### ISSUE THREAD

**Issue by tosiara** [Manouchehri/smi2021#8]
```
user@linux-8sn7:~/smi2021> make -C /lib/modules/$(uname -r)/build M=$PWD modules
make: Entering directory '/usr/src/linux-4.0.3-1-obj/x86_64/desktop'
** GIT MODULE VERSION = b36e2cf **
  CC [M]  /home/user/smi2021/smi2021_main.o
  CC [M]  /home/user/smi2021/smi2021_bootloader.o
  CC [M]  /home/user/smi2021/smi2021_v4l2.o
/home/user/smi2021/smi2021_v4l2.c: In function ‘smi2021_video_register’:
/home/user/smi2021/smi2021_v4l2.c:332:15: error: ‘struct video_device’ has no member named ‘debug’
  smi2021->vdev.debug = 0;   /* enable debug */
               ^
/usr/src/linux-4.0.3-1/scripts/Makefile.build:274: recipe for target '/home/user/smi2021/smi2021_v4l2.o' failed
make[3]: *** [/home/user/smi2021/smi2021_v4l2.o] Error 1
/usr/src/linux-4.0.3-1/Makefile:1416: recipe for target '_module_/home/user/smi2021' failed
make[2]: *** [_module_/home/user/smi2021] Error 2
Makefile:145: recipe for target 'sub-make' failed
make[1]: *** [sub-make] Error 2
Makefile:24: recipe for target '__sub-make' failed
make: *** [__sub-make] Error 2
make: Leaving directory '/usr/src/linux-4.0.3-1-obj/x86_64/desktop'
```

**Comment by tosiara** [comment]
This is openSUSE Tumbleweed 2015-05-16
```
user@linux-8sn7:~/smi2021> cat /etc/issue
Welcome to openSUSE 20150516 "Tumbleweed" - Kernel \r (\l).
user@linux-8sn7:~/smi2021> uname -a
Linux allans-ipad 4.0.3-1-desktop #1 SMP PREEMPT Wed May 13 13:41:19 UTC 2015 (3ee3773) x86_64 x86_64 x86_64 GNU/Linux
```

**Comment by Manouchehri** [comment]
I can confirm it fails here too, I'm running `4.0.4-2-ARCH #1 SMP PREEMPT Fri May 22 03:05:23 UTC 2015`.

```
  CC [M]  /tmp/somagic-easycap-smi2021-git/src/smi2021/smi2021_main.o
  CC [M]  /tmp/somagic-easycap-smi2021-git/src/smi2021/smi2021_bootloader.o
  CC [M]  /tmp/somagic-easycap-smi2021-git/src/smi2021/smi2021_v4l2.o
/tmp/somagic-easycap-smi2021-git/src/smi2021/smi2021_v4l2.c: In function ‘smi2021_video_register’:
/tmp/somagic-easycap-smi2021-git/src/smi2021/smi2021_v4l2.c:333:15: error: ‘struct video_device’ has no member named ‘debug’
  smi2021->vdev.debug = 0;   /* enable debug */
               ^
scripts/Makefile.build:258: recipe for target '/tmp/somagic-easycap-smi2021-git/src/smi2021/smi2021_v4l2.o' failed
make[1]: *** [/tmp/somagic-easycap-smi2021-git/src/smi2021/smi2021_v4l2.o] Error 1
make[1]: *** Waiting for unfinished jobs....
Makefile:1390: recipe for target '_module_/tmp/somagic-easycap-smi2021-git/src/smi2021' failed
make: *** [_module_/tmp/somagic-easycap-smi2021-git/src/smi2021] Error 2
make: Leaving directory '/usr/lib/modules/4.0.4-2-ARCH/build'
```

**Comment by mastervolkov** [comment]
Fixed these in https://github.com/Manouchehri/smi2021/pull/9

**Comment by Manouchehri** [comment]
Compiles fine here now, merged #9. Thanks @mastervolkov.
